providers/os/resources/os.lr

```diff
@@ -1389,6 +1389,9 @@ windows {
 
   // Information about Windows Server roles, role services, and features that are available for installation and installed on a specified server.
   features() []windows.feature
```

Comment:
I think we should do:

```
// Information about Windows Server roles, role services, and features that are available for installation and installed on a specified server.
serverFeatures() []windows.serverfeature

// deprecated: use serverFeature
features() []windows.serverfeature
```

Reply:
I added the new `serverFeature` and deprecated the old one.

Reply:
Awesome.